lib/Sema/TypeCheckStorage.cpp

```diff
@@ -2537,7 +2537,7 @@ RequiresOpaqueAccessorsRequest::evaluate(Evaluator &evaluator,
 
   } else if (dc->isModuleScopeContext()) {
     // Fixed-layout global variables don't require opaque accessors.
-    if (!var->isResilient() && !var->shouldUseNativeDynamicDispatch())
+    if (var->isFragile() && !var->shouldUseNativeDynamicDispatch())
```

Comment:
I'm not sure this is right. Here, the property being resilient forces more accessors to be synthesized, so we want to keep that even if package optimizations are enabled.